### PR TITLE
Task add appropriate pattern for excluding generated proto files from git

### PR DESCRIPTION
**Pull request by RusskovMatvey**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,6 @@
 *.iml
 *.ipr
 
-laundry-management-service/api/src/main/java/com/microservices/laundrymanagementapi/messages/
\ No newline at end of file
+#pattern for excluding generated proto files from git
+*Event.java
+LaundryState.java
```
Comment (AfanasievYuriy) [inline .gitignore]: what about something like this: `**/*api/messages/`? But I also suggest to rename a package to `com.microservices.laundrymanagement.api` and then use `**/api/messages/`

Comment (andrsuh) [reply .gitignore]: Do you want all the messages have same package? It's easier to distinguish which api we use if they have different packages and include service name

Comment (andrsuh) [reply .gitignore]: But I agree We'd better to find some generalized pattern for this

Comment (RusskovMatvey) [reply .gitignore]: i did as AfanasievYuriy said

File: .gitignore [hunk 2]
```diff
@@ -7,4 +7,5 @@
 *.iml
 *.ipr
 
-laundry-management-service/api/src/main/java/com/microservices/laundrymanagementapi/messages/
\ No newline at end of file
+#pattern for excluding generated proto files from git
+**/*api/messages/
```
Comment (AfanasievYuriy) [inline .gitignore]: `**/api/messages/`  but not `**/*api/messages/`- in case you've renamed your package (and you did)

Comment (RusskovMatvey) [reply .gitignore]: fixed

**Review by AfanasievYuriy — DISMISSED** [review]

**Review by andrsuh — APPROVED** [review]

**Pull request merged**